modules/EnsEMBL/Web/QueryStore/Source/Adaptors.pm

```diff
@@ -37,9 +37,15 @@ sub _database {
   my ($self,$species,$db) = @_;
 
   $db ||= 'core';
-  if($db =~ /compara/) {
-    $species = 'multi';
+
+  ## Rapid site has both single-species and multi-species compara
+  if ($self->{_sd}->SINGLE_SPECIES_COMPARA) { 
```

Comment:
How can we test these changes? Can we see in a sandbox? 

Does this affect precache?

Reply:
It's tricky to test right now, because I'm waiting for RR compara dbs to be handed over. The new code here will be ignored by all sites apart from RR, and the changes will be tested on RR before that can go live.

Reply:
So no, no effect on precache, because RR doesn't have it.